.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
do we need it ?

Reply:
we don't need this. it is just vscode settings.